File: chapterseven/src/main/kotlin/Songs.java
import java.util.List;

public class Songs {

    public String addSong(String songName) {
        return "Song added: " + songName;
    }

    public String removeSong(String songName) {
        return "Song removed: " + songName;
    }
    public List<String> getSongList() {
        return List.of("Song 1", "Song 2", "Song 3");
    }
}

File: chaptersix/src/main/kotlin/Songs.kt
class SongsKt {
    fun addSong(songName: String): String {
        return "Song added: $songName"
    }

    fun removeSong(songName: String): String {
        return "Song removed: $songName"
    }

    val songList: List<String>
        get() = listOf("Song 1", "Song 2", "Song 3")
}
